src/main/java/chat/command/FindCommand.java

```diff
@@ -0,0 +1,47 @@
+package chat.command;
+
+import chat.*;
```

Comment:
Perhaps avoid using the wild imports and list imported classes explicitly?

Reply:
Yes that seems better, thank you for pointing that out.